README.md

```diff
@@ -1,6 +1,221 @@
 # Shrinkwrap
 
-ShrinkWrap is a library to create archives from Java, which powers the [Arquillian](https://github.com/arquillian)
-deployment mechanism. Arquillian in turn is a library.
+ShrinkWrap is a Java library used to create Java archives such as JARs, WARs, EARs and RARs.
+This is especially useful for integration testing, where you need to deploy specific configurations of your applications, such as [Arquillian](https://github.com/arquillian).
+
+## Requirements
+
+- **JDK**: Version 8 or newer.
+- **Maven**: Version 2.2.0 or newer.
+
+## Using Shrinkwrap
+
+To include ShrinkWrap in your project, you can add a dependency on the `shrinkwrap-depchain` module in your project's `pom.xml`:
+
+```xml
+  <dependency>
+    <groupId>org.jboss.shrinkwrap</groupId>
+    <artifactId>shrinkwrap-depchain</artifactId>
+    <version>${version.shrinkwrap}</version>
+    <type>pom</type>
+  </dependency>
+```
+
+The `ShrinkWrap` class is the entry point to the ShrinkWrap library, enabling users to create and manage archives and domains.
+It provides static methods to create archives and domains with either default or custom configurations.
+
+### Working With Shrinkwrap
+
+You can create and export JAR files for deployment to Java EE servers or for use in testing frameworks like Arquillian.
+
+```java
+// Create a new Java archive (JAR) using ShrinkWrap.
```

Comment:
The missing line breaks are making this hard to read. In other code blocks there's always an empty line before the comment. I'd use the same approach here

Reply:
Also the indentation is wrong

Reply:
Create a new Java archive (JAR) using ShrinkWrap -> Create a new Java archive (JAR) using ShrinkWrap containing TestClass class and a StringAsset